src/main/resources/db.changelog/db.changelog-master.yaml

```diff
@@ -5,6 +5,22 @@ databaseChangeLog:
       file: changes/init.yaml
       relativeToChangelogFile: true
 
+  - include:
+      file: changes/seat.yaml
+      relativeToChangelogFile: true
+
+  - include:
+      file: changes/equipment.yaml
+      relativeToChangelogFile: true
+
+  - include:
+      file: changes/notification.yaml
+      relativeToChangelogFile: true
+
+  - include:
+      file: changes/em_notif.yaml
+      relativeToChangelogFile: true
+
```

Comment:
I think you should move these lines to the end. Please we may have some conflicts when creating table.